problems/variables.js

```diff
@@ -1,48 +1,78 @@
 // ## Problem One
-
+"use strict"
 // Age Calculator:
-//  * Store your birth year in a constant variable.
+//  * Store your birth year in a constant variable. 
+// const birthYear = 1996;
 //  * Store a future year in a variable.
+// let futureYear = 2021;
 //  * Calculate your 2 possible ages for that year based on the stored values.
+// let age = futureYear - birthYear;
+// console.log ('I will be either' + " " + age + " " + 'or'+ " " +(age +1));
+
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
 
+
 // ## Problem Two
 
 // Snack Supply Calculator:
 //  * Store your current age in a variable.
+ // let age = 24;
 //  * Store a maximum age in a constant variable.
+ // const maxAge = 100;
 //  * Store an estimated snack amount per day (as a number).
+// let numPerDay = 5;
 //  * Calculate how many snacks you would eat total, from your current age until the maximum age.
+// let totalNeeded = ((numPerDay * 365 ) * (maxAge - age));
 //  * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
+// console.log(totalNeeded);
 
 // ## Problem Three
 
 // Calculate properties of a circle, using the definitions: http://math2.org/math/geometry/circles.htm 
 //  * Store a radius into a variable.
```

Comment:
All variables should be let or const. Var is an older more outdated way of writing code. Please use let and const from here on out unless otherwise told to do var.